modules/fhir-structure/src/blaze/fhir/spec.clj

```diff
@@ -115,6 +116,14 @@
       (s2/unform spec resource)
       (throw (ex-info (format "Missing spec: %s" key) {:key key})))))
 
+(defn unform-binary
```

Comment:
I would not create an unform binary at all. Unforming is the transformation from the internal format into JSON or XML, which are both structured formats of the whole resource. Outputting the binary part of the binary resource is just an extraction of a single value from the internal format.

Reply:
I'll remove this last level of indirection.